webapp/src/components/Vault/VaultActionsForm/Modal/ActionSteps.tsx

```diff
@@ -232,6 +233,13 @@ const ActionSteps: React.FC<ActionStepsProps> = ({
                 }
                 break;
 
+              case "rsAVAX-THETA":
```

Comment:
Not urgent, but at some point we should migrate all these asset deposit handlers to a separate file